assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +355,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
```

Comment:
Opened #2163
@spadgett add a TODO to use the annotation to determine whether the host was generated or not